### PR TITLE
Updated dbs and models for support app

### PR DESCRIPTION
**Pull request by johanlundberg**

### CODE REVIEW

File: src/eduid_userdb/support/db.py [hunk 1]
```diff
@@ -75,27 +69,19 @@ def get_authn_info(self, user_id):
...
+        :type credential_id: ObjectId | str | unicode
+        :return:  A document dict
+        :rtype: dict
         """
-        if not isinstance(user_id, ObjectId):
-            user_id = ObjectId(user_id)
-        docs = self._get_documents_by_attr('user_oid', user_id, raise_on_missing=False)
-        return [self.model(doc) for doc in docs]
+        if not isinstance(credential_id, ObjectId):
+            user_id = ObjectId(credential_id)
```
Comment (fredrikt) [inline src/eduid_userdb/support/db.py]: this should be credential_id = ObjectId(credential_id), right?

File: src/eduid_userdb/support/models.py [hunk 2]
```diff
@@ -44,48 +39,29 @@ def __init__(self, data):
...
-        _data_in = DashboardUser(data=data).to_dict()
-        _user_id = _data_in['_id']
-        super(SupportDashboardUser, self).__init__(_data_in)
+        _data = deepcopy(data)
+        super(SupportUserFilter, self).__init__(_data)
 
-        self['user_id'] = _user_id
         self['mailAliases'] = [MailAlias(alias) for alias in self['mailAliases']]
-        self['passwords'] = [Password(password) for password in self['passwords']]
+        self['passwords'] = [Credential(password) for password in self['passwords']]
```
Comment (fredrikt) [inline src/eduid_userdb/support/models.py]: I think you should let userdb parse data into a User() of some sort and then build your dict from the User object. The dict representations of users is subject to change.

Comment (johanlundberg) [reply src/eduid_userdb/support/models.py]: The in data comes from User().to_dict. The GenericFilterDict wont fail on missing data so if the underlying user dict changes we have to update our models but it won't crash.

File: src/eduid_userdb/support/db.py (hunk 1 above)
Comment (johanlundberg) [reply src/eduid_userdb/support/db.py]: Right!

**Pull request merged**